CRM/Core/ClassLoader.php

```diff
@@ -104,7 +104,8 @@ public function register($prepend = FALSE) {
     }
     $civicrm_base_path = dirname(dirname(__DIR__));
 
-    require_once dirname(dirname(__DIR__)) . '/vendor/autoload.php';
+    // @todo: need some way to check if we should register vendor or not
```

Comment:
Just pushed a new commit that checks for the local vendor autoload, and loads it if it's there